pkg/apis/pingcap/v1alpha1/tidbmonitor_types.go

```diff
@@ -132,6 +132,9 @@ type PrometheusSpec struct {
 	// +optional
 	ReserveDays int `json:"reserveDays,omitempty"`
 
+	// +optional
```

Comment:
```suggestion
	// Configuration for `--storage.tsdb.retention.time`, Units Supported: y, w, d, h, m, s, ms.
	// If set to non empty values, it will override the value of `ReserveDays`.
	// +optional
```